src/main/java/lotto/domain/winpolicy/ThreeMatch.java

```diff
@@ -1,14 +1,15 @@
 package lotto.domain.winpolicy;
 
 import lotto.domain.lotto.Lotto;
+import lotto.domain.lotto.LottoNumber;
 
 public class ThreeMatch extends Policy {
   public ThreeMatch() {
     super(5_000);
   }
 
   @Override
-  public Boolean isMatch(Lotto latestWinLotto, Lotto lotto) {
+  public Boolean isMatch(Lotto latestWinLotto, Lotto lotto, LottoNumber bonusNumber) {
     if (latestWinLotto.matchCountOf(lotto).equals(3)) {
       return true;
     }
```

Comment:
```suggestion
    return latestWinLotto.matchCountOf(lotto).equals(3);
```
생각해보니 간결하게 표현하는 것도 좋겠네요